src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -63,7 +63,7 @@
 
     const VERSION = '4.0.0-DEV';
     const VERSION_ID = 40000;
-    const MAJOR_VERSION = 4;
+    const MAJOR_VERSION = 0;
```

Comment:
Why did you change this?

Reply:
That seems like a bug in the release script 😛

Reply:
It was already fixed in https://github.com/symfony/symfony/commit/ce519b6e444abb12174fbd57ec0936ea4961ab56